modules/project_cleanup/function_source/main.go

```diff
@@ -235,6 +247,29 @@ func invoke(ctx context.Context) {
 		}
 	}
 
+	removeFirewallPolicies := func(folder string) {
+		logger.Printf("Try to remove Firewall Policies from folder [%s]", folder)
+		firewallPolicyList, err := firewallPoliciesService.List().ParentId(folder).Context(ctx).Do()
+		if err != nil {
+			logger.Printf("Fail to list Firewall Policies from folder [%s], error [%s]", folder, err.Error())
+			return
+		}
+		for _, policy := range firewallPolicyList.Items {
+			for _, association := range policy.Associations {
+				_, err := firewallPoliciesService.RemoveAssociation(policy.Name).Name(association.Name).Context(ctx).Do()
+				if err != nil {
+					logger.Printf("Fail to Remove Association for Firewall Policies from folder [%s], error [%s]", folder, err.Error())
+					return
+				}
+			}
+			_, err := firewallPoliciesService.Delete(policy.Name).Context(ctx).Do()
+			if err != nil {
+				logger.Printf("Fail to delete Firewall Policy [%s] from folder [%s], error [%s]", policy.Name, folder, err.Error())
+				return
```

Comment:
Same as above, cant we continue execution if a delete failed?

Reply:
fixed